tests/pipelines/dance_diffusion/test_dance_diffusion.py

```diff
@@ -73,7 +73,7 @@ def test_dance_diffusion(self):
         audio_from_tuple_slice = audio_from_tuple[0, -3:, -3:]
 
         assert audio.shape == (1, 2, self.dummy_unet.sample_size)
-        expected_slice = np.array([-0.7265, 1.0000, -0.8388, 0.1175, 0.9498, -1.0000])
+        expected_slice = np.array([-0.8682, 1.0000, -1.0000, -0.3130, 0.5133, -1.0000])
```

Comment:
Let's please not change the hard-coded values here. This probably needs to be tested on a V100 . Happy to test for you if the PR is ready

Reply:
I changed them back to original, I guess my question is then why do the tests fail on GH actions? Do we have to indicate them to run on a100 on github sometimes?